nservicebus/dependency-injection/index_ownership_core_[,7.2).partial.md

```diff
@@ -0,0 +1,5 @@
+NServiceBus automatically registers and invokes message handlers, sagas, and other user provided extension points using a dependency injection container.
```

Comment:
This line is common to both 7.1. and 7.2 so it should really be in `index.md`.

Reply:
Agree, I'll move

Reply:
Fixed in 548dda2b48cae3b7e0d8a9776d7bdb1927960d6b